CaptureSDK/GiniCaptureSDK/Tests/GiniCaptureSDKTests/DecimalExtensionTest.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  File.swift
+//  
+//
+//  Created by David Vizaknai on 04.04.2023.
+//
+
+import XCTest
+
+class DecimalExtensionTests: XCTestCase {
+    func testStringValueFromTwoDecimals() {
+        let decimalValue: Decimal = 8.28
+        let decimalString = decimalValue.stringValue(withDecimalPoint: 2)
+        XCTAssertEqual(decimalString, "8.28", "Decimal string value is not equal to expected string value.")
+    }
+
+    func testStringValueFromWholeNumber() {
+        let decimalValue: Decimal = 5
+        let decimalString = decimalValue.stringValue(withDecimalPoint: 2)
```

Comment:
[Suggestion]@vizaknaidavid Could you please add more test cases here for 
1000.09 ?